src/blocks/author-carousel/components/inspector/LayoutPanel.js

```
@@ -0,0 +1,156 @@
+ * @param {Object}   props               Component props
+ * @param {Object}   props.attributes    Block attributes
+ * @param {Function} props.setAttributes Set attributes function
+ * @return {JSX.Element} Layout panel
+ */
+export default function LayoutPanel( { attributes, setAttributes } ) {
+	const {
+		layout,
+		textAlign,
+		padding,
+		slideSpacing,
+		enableShadow,
```

ESLint

[error] 'enableShadow' is assigned a value but never used.

(no-unused-vars)